HelpSource/Classes/Array.schelp

```diff
@@ -57,37 +57,47 @@ copymethod:: ArrayedCollection *series
 
 copymethod:: ArrayedCollection *iota
 
-copymethod:: SequenceableCollection *interpolation
+copymethod:: Collection *interpolation
 
-copymethod:: SequenceableCollection *rand
-
-copymethod:: SequenceableCollection *rand2
-
-copymethod:: SequenceableCollection *linrand
-
-copymethod:: SequenceableCollection *exprand
-
-copymethod:: SequenceableCollection *fib
+copymethod:: Collection *rand
 
+copymethod:: Collection *rand2
 
+copymethod:: Collection *linrand
 
+copymethod:: Collection *exprand
 
+copymethod:: Collection *fib
```

Comment:
I think you have to do it by hand here... git can revert a whole commit. It can't revert changes to one file that is part of a commit. Your first commit includes changes to a lot of files, and there's no way to separate them after the fact.

Generally one commit should be for one set of closely related changes. Then, you can put several commits together into one branch, for a PR.

You have a bunch of unrelated changes in one commit. Probably not the best way.

It might be better to fix the problems, then delete this branch and start over with the PR for a cleaner history.